ehr/resources/web/ehr/form/field/ProjectEntryField.js

```diff
@@ -10,351 +10,352 @@
  * @cfg includeDefaultProjects defaults to true
  */
 Ext4.define('EHR.form.field.ProjectEntryField', {
-    extend: 'Ext.form.field.ComboBox',
-    alias: 'widget.ehr-projectentryfield',
-
-    fieldLabel: 'Project',
-    typeAhead: true,
-    forceSelection: true, //NOTE: has been re-enabled, but it is important to let the field get set prior to loading
-    emptyText:'',
-    disabled: false,
-    matchFieldWidth: false,
-    includeDefaultProjects: true,
-    invesLastNameCol: 'lastName',
```

Comment:
This looks like a problem, as TNPRC is overriding this value and this change appears to completely snip out the investigator value from the template. We should coordinate with WNPRC on the motivation for this change.

Reply:
I've restored invesLastNameCol for now, will sync with WNPRC

Reply:
Email sent to client about invesLastNameCol.